src/capi-single-paidfor/web/index.scss

```diff
@@ -1,15 +1,57 @@
-@import '_core';
-@import '_adverts';
-@import '_advert';
-@import '_adverts-capi';
-@import '_paidfor-meta';
-@import '_popup-paidfor';
+@import "_core";
+@import "_adverts";
+@import "_advert";
+@import "_adverts-capi";
+@import "_paidfor-meta";
+@import "_popup-paidfor";
 
 .paidfor-meta__more .paidfor-meta__label {
-    font-weight: normal;
+  font-weight: normal;
+}
+
+.adverts--legacy-single.adverts--capi.adverts--paidfor {
+  .adverts__row,
+  .adverts__body {
+    padding: 0;
+  }
+
+  .advert {
+    @include mq(mobileLandscape) {
+      flex-grow: 1;
+    }
+  }
+
+  .advert__image-container {
+    flex-basis: 100%;
+    height: 100%;
+
+    @include mq(tablet) {
+      flex-basis: 500px;
+      flex-grow: 0;
+    }
+  }
+
+  .advert__text {
+      justify-content: space-between;
+      flex-grow: 1;
+  }
+
+  .badge {
+    width: 200px;
+    text-align: left;
+
+    &::before {
+      display: none;
+    }
```

Comment:
This seems a bit weird? Or am I missing some CSS trick?

Reply:
You _are_ missing a trick: it's SASS.

The `&` is replaced by the parent item when converted to CSS. Here we could simplify it if we're not adding any other rule…

Reply:
Yeah I understand its SASS but why the `display:none` on `::before` ?

Reply:
It's overriding a rule that displays a line…